inst/template/mode.cpp

```diff
@@ -11,3 +11,8 @@ SEXP mode_{{name}}_alloc(cpp11::list r_pars, double time) {
 double mode_{{name}}_time(SEXP ptr) {
   return mode::r::mode_time<mode::container<{{class}}>>(ptr);
 }
+
+[[cpp11::register]]
+std::vector<double> mode_{{name}}_solve(SEXP ptr, int end_time) {
+return mode::r::mode_solve<mode::container<{{class}}>>(ptr, end_time);
```

Comment:
```suggestion
  return mode::r::mode_solve<mode::container<{{class}}>>(ptr, end_time);
```

I expect your editor will struggle to format this file as it'll be syntactically invalid